tsconfig.json

```
@@ -33,5 +33,6 @@
     }
```

```suggestion
    "Types/*": ["types/*"]
    }
```

You need to add this path